ledger-tool/src/main.rs

```diff
@@ -1079,10 +1086,15 @@ fn main() {
                 .arg(
                     Arg::with_name("record_slots_config")
                         .long("record-slots-config")
-                        .default_value("hash-only")
-                        .possible_values(&["hash-only", "accounts"])
+                        .multiple(true)
+                        .takes_value(true)
+                        .possible_values(&["accounts", "tx"])
                         .requires("record_slots")
-                        .help("In the slot recording, include bank details or not"),
+                        .conflicts_with_all(&[
+                            "enable_rpc_transaction_history",
+                            "geyser_plugin_config",
+                        ])
+                        .help("In the slot recording, include bank account or transaction details"),
```

Comment:
nit: What do you think about this tweak, this helps imply the default behavior of writing just the bank hash + the other items being optional add-ons (that can both be used)
```rust
.help("In addition to the bank hash, optionally include accounts and/or transactions details for the slot"),
```

Reply:
Agreed, that is better